dask/dataframe/core.py

```diff
@@ -7819,9 +7820,12 @@ def maybe_shift_divisions(df, periods, freq):
 @wraps(pd.to_datetime)
 def to_datetime(arg, meta=None, **kwargs):
     tz_kwarg = {"tz": "utc"} if kwargs.get("utc") else {}
+
+    xd = get_meta_library(arg)
```

Comment:
`dask/dataframe/tests/test_dataframe.py::test_to_datetime` is failing here. It looks like we might need to move this line of code around